content/en/docs/tutorials/configuration/updating-configuration-via-a-configmap.md

```diff
@@ -0,0 +1,381 @@
+---
+title: Updating Configuration via a Configmap
+content_type: tutorial
+weight: 20
+---
+
+<!-- overview -->
+This page provides a step-by-step example of updating configuration within a Pod via a ConfigMap and builds upon the [Configure a Pod to Use a ConfigMap](/docs/tasks/configure-pod-container/configure-pod-configmap/) task. At the end of this tutorial, you will understand how to change the configuration for a running application. 
+This tutorial uses the NGINX image as an example.
+
+## {{% heading "prerequisites" %}}
+{{< include "task-tutorial-prereqs.md" >}}
+
+## {{% heading "objectives" %}}
+* Update configuration via a ConfigMap mounted as a Volume
+* Update environment variables of a Pod via a ConfigMap 
+
+<!-- lessoncontent -->
+
+## Update configuration via a ConfigMap mounted as a Volume
+
+Use the `kubectl create configmap` command to create a ConfigMap from [literal values](/docs/tasks/configure-pod-container/configure-pod-configmap/#create-configmaps-from-literal-values):
+
+```shell
+kubectl create configmap sport --from-literal=sport=football
+```
+Below is an example of a Deployment manifest with the ConfigMap `sport` mounted as a Volume on the Pod.
+{{% code_sample file="deployments/deployment-with-configmap-as-volume.yaml" %}}
+
+Create the deployment.
+```shell
+kubectl apply -f https://k8s.io/examples/deployments/deployment-with-configmap-as-volume.yaml
+```
+
+Check the deployment.
+```shell
+kubectl get deployment configmap-volume
+```
+
+You should see an output similar to:
+```
+NAME               READY   UP-TO-DATE   AVAILABLE   AGE
+configmap-volume   3/3     3            3           19s
+```
+
+Check the pods.
+```shell
+kubectl get pods --selector=app=configmap-volume
+```
+
+You should see an output similar to:
+```
+NAME                                READY   STATUS    RESTARTS   AGE
+configmap-volume-6b976dfdcf-qxvbm   1/1     Running   0          72s
+configmap-volume-6b976dfdcf-skpvm   1/1     Running   0          72s
+configmap-volume-6b976dfdcf-tbc6r   1/1     Running   0          72s
+```
+
+The data in the ConfigMap is mounted to the Pods as a Volume. The Pod references this data to generate an output that is printed to stdout. Check this by viewing the logs of the Pods.
+```shell
+kubectl logs deployments/configmap-volume
```

Comment:
```suggestion
# Pick one Pod that belongs to the Deployment, and view its logs
kubectl logs deployments/configmap-volume
```

Reply:
This should not be necessary as `kubectl logs deployments/configmap-volume` picks one of the Pod at random and displays its logs. This is seen in the output of the command, similar to `Found 3 pods, using pod/configmap-volume-76d9c5678f-x5rgj`